protobuf-maven-plugin/src/main/java/io/github/ascopes/protobufmavenplugin/mojo/AbstractGenerateMojo.java

```diff
@@ -347,6 +347,17 @@ public AbstractGenerateMojo() {
   @Parameter(defaultValue = DEFAULT_FALSE)
   boolean fatalWarnings;
 
+  /**
+   * Defines the file in which to write the protobin descriptor.
+   *
+   * <p>Leave unspecified to disable. Writes a FileDescriptorSet (a protocol buffer,
+   * defined in descriptor.proto) containing all the input files in {@code descriptorFile}.</p>
+   *
+   * @since 2.7.4
```

Comment:
```suggestion
   * @since 2.9.0
```

I released 2.8.0 just as you raised this PR!